requirements/base.txt

```diff
@@ -20,6 +20,7 @@ django-tinymce==2.2.0
 django-treebeard==3.0
 elasticsearch==1.6.0
 feedparser==5.2.1
+flake8==3.0.4
```

Comment:
flake8 shouldn't be here, it is already in [requirements/test.txt](https://github.com/cfpb/cfgov-refresh/blob/master/requirements/test.txt#L6).